Cargo.lock

```diff
@@ -504,6 +504,21 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "alloy-rpc-types-engine"
+version = "0.3.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1c031a91e94a39f928244bc837c953817be5b8cc61759e1a9123b3abd17560dd"
+dependencies = [
+ "alloy-consensus",
+ "alloy-eips",
+ "alloy-primitives",
+ "alloy-rlp",
+ "alloy-serde",
+ "derive_more 1.0.0",
+ "serde",
+]
```

Comment:
The issue is in upstream `alloy-rpc-types-engine` as well. Working on fixing this in pr https://github.com/alloy-rs/alloy/pull/1284